src/r/DESCRIPTION

```diff
@@ -0,0 +1,26 @@
+Package: psychrolib
+Type: Package
+Title: Psychrometric Properties of Moist and Dry Air
+Version: 2.3.0
+Authors@R: c(person("Hongyuan", "Jia", email = "hongyuan.jia@bears-berkeley.sg", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-0075-8183")),
+             person("Jason", "Banfelder", email = "jason@banfelder.net", role = "aut"),
+             person("D.", "Meyer", role = "aut"),
+             person("D.", "Thevenard", role = "aut"))
+Description:
+    A port of the original python 'psychrolib'
+    (<https://github.com/psychrometrics/psychrolib>) library written by D. Meyer &
+    D. Thevenard, which provides functions to enable the calculation of
+    psychrometric properties of moist and dry air. The library works in both
+    metric (SI) and imperial (IP) systems of units.
+License: MIT + file LICENSE
```

Comment:
Can we not just include one file?

Reply:
In terms of CRAN package convetions, please see my previous comments https://github.com/psychrometrics/psychrolib/pull/49#issuecomment-568298693.

Specifically, for `License` field, see https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Licensing

Basically, `MIT` here does not indicate a file but a license type.

Reply:
Don't you think this is confusing though? If you click on MIT it takes to the MIT licence, if you click on LICENSE, it will take you to ours (the one you copy and upload). Has people not complained about this before in the R community?

Reply:
It's quite clear IMO. Haven't seen any people complained about that.

Reply:
@dmey, I think I find the answer for your confusing.

Based on https://www.r-project.org/Licenses/MIT, the only contents should be put in the `LICENSE` would be

```
YEAR:
COPYRIGHT HOLDER: 
```

That's why I get a [NOTE](https://win-builder.r-project.org/7Hj8MYHo1dre/00check.log) when checking with http://win-builder.r-project.org/.
```
License components with restrictions and base license permitting such:
  MIT + file LICENSE
File 'LICENSE':
  The MIT License (MIT)
  
  Copyright (c) 2018 D. Thevenard and D. Meyer.
  
 ...
```
I will update `deploy.R` to make sure only those 2 fields are copied.

Reply:
I have update the licence -- could you please modify so that the licence file is created with the following please:

```
Copyright (c) 2018-2020 The PsychroLib Contributors.

```

Reply:
@dmey Great! I have updated `deploy.R` to support this format.